config/vmagent-prometheus.yml.erb

```diff
@@ -0,0 +1,15 @@
+global:
```

Comment:
It might be useful to add an `external_labels` section. These are labels that will be added to any timeseries.

This could be useful for setting things like environment, region, dyno etc. 

It would mean that you wouldn't have to set the dyno label twice on each individual scrape target.

Perhaps ERB templating makes this difficult.

Reply:
Ah nice, thanks! I've moved the dyno labels up to `external_labels`, and also added support for user-defined labels via the `VMAGENT_EXTERNAL_LABELS` config var: bd6909c